packages/views/search.py

```diff
@@ -151,6 +177,36 @@ def get_context_data(self, **kwargs):
         context['search_form'] = self.form
         return context
 
+def group_search_json(request) -> HttpResponse:
+    limit = 250
+
+    container = {
+        'version': 2,
+        'limit': limit,
+        'valid': False,
+        'results': [],
+    }
+
+    if request.GET:
+        form = GroupSearchForm(data=request.GET,
+                                 show_staging=request.user.is_authenticated)
+
+        if form.is_valid():
+            form_limit = form.cleaned_data.get('limit', limit)
+            limit = min(limit, form_limit) if form_limit else limit
+            container['limit'] = limit
```

Comment:
these two lines can be combined into one

Reply:
Stole it from the package search hehe, I'll combine it :)